includes/admin/views/setup-wizard/step-pages.php

```diff
@@ -4,16 +4,17 @@
  *
  * @since 4.4.4
  * @since 7.3.0 Using the `LLMS_Install::get_pages()` method now.
- * @version 7.3.0
+ * @since [version] Escape remaining strings.
+ * @version [version]
  *
  * @property LLMS_Admin_Setup_Wizard $this Setup wizard class instance.
  */
 
 defined( 'ABSPATH' ) || exit;
 ?>
-<h1><?php _e( 'Page Setup', 'lifterlms' ); ?></h1>
+<h1><?php esc_html_e( 'Page Setup', 'lifterlms' ); ?></h1>
```

Comment:
there's a printf at the end of this file:
```<p><?php printf( __( 'After setup, you can manage these pages from the admin dashboard on the %1$sPages screen%2$s and you can control which pages display on your menu(s) via %3$sAppearance > Menus%4$s.', 'lifterlms' ), '<a href="' . esc_url( admin_url( 'edit.php?post_type=page' ) ) . '" target="_blank">', '</a>', '<a href="' . esc_url( admin_url( 'nav-menus.php' ) ) . '" target="_blank">', '</a>' ); ?></p>```

This:
`__( 'After setup, you can manage these pages from the admin dashboard on the %1$sPages screen%2$s and you can control which pages display on your menu(s) via %3$sAppearance > Menus%4$s.', 'lifterlms' )`

needs to be escaped as well, and suggestions to translators (because of the placeholders) should be added.
can you?